spec/graphql/types/query_type_spec.rb

```diff
@@ -35,57 +65,83 @@ def todos_query
       create(:todo, name: 'task3')
     end
 
+    after(:each) do
+      if parsed_errors
+        puts "GraphQL Errors: #{parsed_errors.first['message']}"
+      end
+    end
```

Comment:
this is used to display the errors in console, why do we need this?
if we want to see errors we should have specific tests that asserts those errors